packages/nimbus/src/components/icon-button/icon-button.mdx

```diff
@@ -163,20 +163,20 @@ const App = () => {
 
 #### Tooltip
 
-For accessibility compliance (and good user experience for all) all icon buttons
-must have a tooltip. Go to tooltips for more formation about the component.
+For accessibility compliance (and good user experience for all) icon buttons
+that do not have a label must have a tooltip. Go to tooltips for more information about the component.
```

Comment:
We use icon buttons in the combobox and elsewhere without a tooltip (toggle buttons for popovers mostly), but they are labelled correctly in the context of the component, so I figured we could clarify that tooltips are for unlabelled/stand-alone icon buttons.